engine/src/main/java/org/terasology/rendering/dag/nodes/RefractiveReflectiveBlocksNode.java

```diff
@@ -179,6 +181,7 @@ public RefractiveReflectiveBlocksNode(Context context) {
         addDesiredStateChange(new SetInputTexture2D(textureSlot++, "engine:waterNormalAlt", CHUNK_MATERIAL_URN, "textureWaterNormalAlt"));
         addDesiredStateChange(new SetInputTextureFromFbo(textureSlot++, REFLECTED_FBO_URI, ColorTexture, displayResolutionDependentFBOs, CHUNK_MATERIAL_URN, "textureWaterReflection"));
         addDesiredStateChange(new SetInputTextureFromFbo(textureSlot++, lastUpdatedGBuffer, ColorTexture, displayResolutionDependentFBOs, CHUNK_MATERIAL_URN, "texSceneOpaque"));
+        addDesiredStateChange(new SetInputTextureFromFbo(textureSlot++, COPY_DEPTH_FBO_URI, DepthStencilTexture, displayResolutionDependentFBOs, CHUNK_MATERIAL_URN, "texSceneOpaqueDepth"));
```

Comment:
Hmmmm... what are you doing here? Is this PR about adding new nodes to the engine or addressing the issue of water look? I'd keep it to the first task only: once the new nodes are available you can then use them in a separate PR about the water look.